circle.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:4
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: $ npm install
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: catch-uncommitted
+          command: npm run cu
```

Comment:
We don't have a template for that, but I'd prefer if we used the resin-sdk approach (for more consistency) by having a 

` "ci": "npm test && catch-uncommitted",`

npm script and have a single `npm run ci` circleci `run` action here

See: https://github.com/resin-io/resin-sdk/blob/master/package.json#L32

Reply:
fixed.